src/com/google/javascript/jscomp/js/es6/util/inherits.js

```diff
@@ -49,25 +49,31 @@
  *
  * @param {!Function} childCtor Child class.
  * @param {!Function} parentCtor Parent class.
+ * @param {boolean} es5Out language out is ES5
  */
-$jscomp.inherits = function(childCtor, parentCtor) {
-  /** @constructor */
-  function tempCtor() {}
-  tempCtor.prototype = parentCtor.prototype;
-  childCtor.superClass_ = parentCtor.prototype;
-  childCtor.prototype = new tempCtor();
-  /** @override */
-    childCtor.prototype.constructor = childCtor;
+$jscomp.inherits = function(childCtor, parentCtor, es5Out) {
+  if (es5Out === true && Object.setPrototypeOf) {
```

Comment:
@shicks 
Hmmm... Sounds like this may be a first use case for environment-specific targeting.

Either we could add a new language_out value for ES5_WITH_SET_PROTOTYPE (or some much better name), or we could add a flag for "make static inheritance work, I know the results won't work right on browsers without setPrototypeOf".